spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/data/SparkParquetWriters.java

```diff
@@ -316,6 +323,35 @@ public void write(int repetitionLevel, Decimal decimal) {
     }
   }
 
+  private static PrimitiveWriter<UTF8String> uuids(ColumnDescriptor desc) {
```

Comment:
The other factory methods are grouped at the top just under the visitor. Should we move this there as well?

Reply:
yep, done